src/tests/class_tests/openms/source/SpectrumExtractor_test.cpp

```diff
@@ -45,6 +46,232 @@ using namespace std;
 START_TEST(SpectrumExtractor, "$Id$")
 
 /////////////////////////////////////////////////////////////
+// Dummy peak data
+
+MSSpectrum spectrum;
+spectrum.resize(73);
+MSSpectrum::Iterator it = spectrum.begin();
+
+it->setMZ(61.92);
```

Comment:
Add a note on where this data is coming from in a multiline comment
// Raw spectrum data acquired in DDA mode (i.e., product ion full spectrum scan)
// measured on a QTRAP 5500 corresponding to C-Aconitate
// taken from E. coli grown on glucose M9 during steady-state 
// for flux analysis.